CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # \[Unreleased\]
 
+* **BREAKING**: the `hdi` crate has been renamed to `hdi`
```

Comment:
Isn't it the holochain_deterministic_integrity crate which has been renamed?

Reply:
thank you for catching this. i ran the rename command again after writing the change log so it was subject to the rename :laughing: